src/includes/getting-started-primer/react-native.mdx

```diff
@@ -0,0 +1,20 @@
+<Note>
+
+_Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies perfomance issues in your application. The React Native SDK includes performance monitoring (our `@sentry/tracing` package) by default._
+
+</Note>
+
+Features:
```

Comment:
I really like the feature list as the first thing people see. It's super helpful to those landing here without understanding well what Sentry can do just yet and should do a good job selling them on moving on with installing.